### PR TITLE
Added README_PTBR.md + links to it

### PR DESCRIPTION
**Pull request by TsukiGva2**
Added Portuguese as a language option, along with the necessary translations.

Please consider merging this pull request, if
you have any concerns, feel free to reach out
via a comment. This is my first actual pull
request in a repo, make me feel useful! XD

### CODE REVIEW

File: README_PTBR.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        </tr>
+        <tr>
+            <td><a href="https://github.com/eust-w/rsm">rsm</a> - Uma aplicação que agrega diversas funções para a utilização de feeds RSS</td>
+            <td><a href="https://github.com/eust-w/obsidian-image-auto-upload">obsidian image plugin</a> - Plugin para colar imagens da área de transferência</td>
+        </tr>
+        <tr>
+            <td><a href="https://github.com/eust-w/qcow2file">qcow2file</a> - Utensílio para gerar imagens `qcow2` a partir de um arquivo docker</td>
+            <!--- 404?
+            <td><a href="https://github.com/eust-w/aiPlatform">ai with wechat</a> - WeChat chat robot uses various AI</td>
+            -->
```
Comment (TsukiGva2) [inline README_PTBR.md:31]: This page doesn't seem to exist, i can still translate it if you ask me to.

File: README_PTBR.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<div align="center">
+    <h3>Hi 👋, I'm <a href="https://longtao.fun">Longtao</a></h3>
```
Comment (TsukiGva2) [inline README_PTBR.md:2]: Forgot to translate this part, this is simply
`<h3>Olá 👋, eu me chamo <a href="https://longtao.fun">Longtao</a></h3>`

File: README_PTBR.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    <p align="center">
+        <a href="https://github.com/eust-w/eust-w/blob/main/README_CN.md"><span>中文</span></a>|
+        <a href="https://github.com/eust-w/eust-w/blob/main/README_FR.md"><span>Français</span></a>|
+        <a href="https://github.com/eust-w/eust-w/blob/main/README_RU.md"><span>Французский</span></a>|
+        <a href="https://github.com/eust-w/eust-w/blob/main/README_AR.md"><span>عربي</span></a>|
+        <a href="https://github.com/eust-w/eust-w/blob/main/README_JP.md"><span>日本語</span></a>|
+        <a href="https://github.com/eust-w/eust-w/blob/main/README_PTBR.md"><span>Português</span></a>|
+    </p>
+    <p>🌟 Seja bem vindo ao meu cantinho no mundo da programação! 🌟</p>
+    <h4 align="left">My Open Source Projects</h4>
```
Comment (TsukiGva2) [inline README_PTBR.md:17]: This one too, `<h4 align="left">Meus projetos _open source_</h4>`

**Comment by eust-w** [comment]
LGTM

**Pull request merged**